Add proxy conf example for mytinytodo

breakall opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

<!--- Before submitting a pull request please check the following -->

<!---  That you have made a branch in your fork, we'd rather not merge from your master -->
<!---  That if the PR is addressing an existing issue include, closes #<issue number> , in the body of the PR commit message   -->
<!---  You have included links to any files / patches etc your PR may be using in the body of the PR commit message -->
<!---  -->

##  Thanks, team linuxserver.io

aptalca commented on mytinytodo.subfolder.conf.example:
```diff
@@ -0,0 +1,12 @@
+# works with https://github.com/breakall/mytinytodo-docker
+# set the mtt_url config in db/config.php
```
Perhaps should read `# set the mtt_url config to todo in db/config.php`?

breakall commented:
You need to enter the full URL that serves as the base for the mytinytodo installation into the value for that config setting. 

e.g. ‘https://sub.domain.com/todo/‘

The key is that the user should go into that config file and fill out that setting according to the instructions. 

> On Mar 13, 2019, at 7:42 PM, aptalca <notifications@github.com> wrote:
> 
> @aptalca commented on this pull request.
> 
> In mytinytodo.subfolder.conf.example:
> 
> > @@ -0,0 +1,12 @@
> +# works with https://github.com/breakall/mytinytodo-docker
> +# set the mtt_url config in db/config.php
> Perhaps should read # set the mtt_url config to todo in db/config.php?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

breakall commented:
> Please add the optional auth bits for ldap and http auth like here:
> https://github.com/linuxserver/reverse-proxy-confs/blob/master/airsonic.subfolder.conf.sample#L4-L10
> 
> And make sure to add the resolver as well: https://github.com/linuxserver/reverse-proxy-confs/blob/master/airsonic.subfolder.conf.sample#L13

Will do.

breakall commented:
I'm new to contributing to other projects on github. I've changed the code in my fork addressing your review notes-- please let me know if I need to do anything else.

aptalca commented:
No worries, you're doing it the right way with the edits.

Please move the auth lines into the second location block like the example in the link.

With regards to the info at the top, as a general rule, be as specific as possible. For instance, the conf expects the sub folder (url base) in mytinytodo to be set to /todo and if it's set to something else it won't work. You need to specify that otherwise people will set theirs to mytinytodo or whatever they feel like and then come here to say it's not working.
